docs/pages/settings-gentime.md

```diff
@@ -70,3 +70,13 @@ settings.change({
 ```
 
 It is considered idiomatic to use the Nexus Prisma configuration file instead of inline generator block configuration. Inline generator block configuration lacks autocomplete and inline JSDoc. The only reason `output` is supported is to be [symmetrical with Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-location-of-prisma-client) and thus ease onboarding.
+
+If you added a custom output path because the output will be in `.js` you will need to include the `.js` files into your `tsconfig.ts` file by adding this prop.
```

Comment:
I know I wrote this code before but I forget why do we need to output JS and not just TS?

Reply:
Do you mean when we have a custom output into the project source, why we don't output TS?

Reply:
Yes

Reply:
That makes us build two builders, one to generate `.js`, `d.ts` by default for `node_modules` and another one for creating a `TS` file if we have a custom output